layer/main.py

```diff
@@ -468,17 +469,20 @@ def start_train(
         train.save_model(trained_model)
 
     """
-    project_name = current_project_name()
-    if not project_name:
-        raise Exception("Missing current project name")
+    project_full_name = current_project_full_name()
+    if not project_full_name:
+        raise Exception(
+            "Missing current project name, please use 'set_current_project_full_name(\"account-name/project-name\")'"
```

Comment:
nit: I think we should only suggest using layer.init(). 
feel free to skip this comment if this PR needs to be merged to unblock things.

Reply:
good shout

Reply:
fixed